src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -491,7 +491,8 @@ let ``Shadows --shots`` () =
 
 /// The expected output from the resources estimator.
 let private resourceSummary =
-    "Metric          Sum Max
+    "The local Resources Estimator will be removed in the March 2023. Use the new Azure Resource Estimator.
```

Comment:
Wording: "in March 2023" sounds more natural.

Reply:
The second part could also be something like: "The Resources Estimator is now available through Azure Quantum."

Reply:
Thank you! Updated message.
'The local Resources Estimator will be removed in March 2023. The Resources Estimator is now available through Azure Quantum.'